template/workflows/helm/.github/workflows/azure-kubernetes-service-helm.yml

```
@@ -44,8 +45,9 @@ env:
   ACR_RESOURCE_GROUP: {{ .Config.GetVariableValue "ACRRESOURCEGROUP" }}
   AZURE_CONTAINER_REGISTRY: {{ .Config.GetVariableValue "AZURECONTAINERREGISTRY" }}
   CONTAINER_NAME: {{ .Config.GetVariableValue "CONTAINERNAME" }}
-  CLUSTER_RESOURCE_GROUP: {{ .Config.GetVariableValue "CLUSTERRESOURCEGROUP" }}
   CLUSTER_NAME: {{ .Config.GetVariableValue "CLUSTERNAME" }}
+  CLUSTER_RESOURCE_GROUP: {{ .Config.GetVariableValue "CLUSTERRESOURCEGROUP" }}
+  CLUSTER_RESOURCE_TYPE: Microsoft.ContainerService/{{ .Config.GetVariableValue "CLUSTERRESOURCETYPE" }}
```

Why can't we do the following?

```suggestion
  CLUSTER_RESOURCE_TYPE: {{ .Config.GetVariableValue "CLUSTERRESOURCETYPE" }}
```

---

We can move the prepend to Devhub, however the resource type discerned from the AKS resource ID is returned as "fleets" or "managedClusters" in Devhub. If I prepend the "Microsoft.ContainerService" to "fleets" or "managedClusters" in the config passed to draft, this will make the resource type consistently have "Microsoft.ContainerService" prepended across all workflow files.